tgui/packages/tgui/interfaces/CommandTablet.js

```diff
@@ -64,10 +70,23 @@ export const CommandTablet = (_props, context) => {
                     evacuation procedures.
                   </NoticeBox>
                 )}
+                <Flex.Item>
+                  <Button.Confirm
+                    fluid={1}
+                    color="orange"
+                    icon="phone-volume"
+                    content={"Send Distress Beacon"}
+                    confirmColor="bad"
+                    confirmContent="Confirm?"
+                    confirmIcon="question"
+                    onClick={() => act('distress')}
+                    disabled={!canDistress} />
```

Comment:
needs to tell the user why distress is disabled; is it cooldown, alert level, etc